code/modules/client/verbs/ooc.dm

```diff
@@ -112,6 +112,7 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 			discordsendmsg("ooc", "[discord_ooc_tag](OOC) **[sayer]:** [msg]")
 		if(CHAT_TYPE_DEADCHAT) // don't send these until a round is finished
 			if(SSticker.current_state == GAME_STATE_FINISHED)
+				msg = copytext(msg, 24, length(msg)-7) // wipes some bad dchat format
```

Comment:
this seems really ugly and bound to break, it should use regex to parse out what it wants instead

Reply:
simple is best. If it is broken in the future, a regex will be broken as well, as a regex for this isn't very nice at all.